queries/highlights.scm

```diff
@@ -11,8 +11,7 @@
   name: (field_identifier) @function)
 
 (template_function
-  name: (scoped_identifier
-    name: (identifier) @function))
+  name: (identifier) @function)
```

Comment:
Queries will be easier to define now that template_function, template_method, and template_type no longer have nested scoped types for the actual type. It's simply the rightmost (non-scoped) identifier.